src/specific/s_filesystem.c

```diff
@@ -1,19 +1,42 @@
 #include "specific/s_filesystem.h"
 
 #include "log.h"
+#include "memory.h"
 
 #include <SDL2/SDL_filesystem.h>
 #include <assert.h>
+#include <dirent.h>
 #include <string.h>
 
 #if defined(_WIN32)
     #include <direct.h>
+    #define strcasecmp _stricmp
+    #define strncasecmp _strnicmp
```

Comment:
Is this really needed? Especially given our docker toolchain which is fairly posix compliant?

Reply:
This one I'm not 100% sure of. If I remove it, it still builds and runs for me. So, I guess the answer is yes.